rest-api/tools/deploy_app.sh

```diff
@@ -100,6 +106,12 @@ fi
 echo "${BOLD}Checking out code...${NONE}"
 git checkout $VERSION
 
+if [[ $(git status --porcelain) ]]; then
```

Comment:
Can we add one more just before gcloud app deploy?

Reply:
done